src/core/utils.py

```diff
@@ -0,0 +1,21 @@
+import open3d as o3d
+
+from src.core.voxel_grid import VoxelGrid
+
+
+def voxel_down_sample(
+    point_cloud: o3d.t.geometry.PointCloud,
+    voxel_grid: VoxelGrid,
+) -> o3d.t.geometry.PointCloud:
+    """
+    Voxel down sampling with voxel grid given
```

Comment:
Please, remain comment somewhere why do we need this function (say about updates in 0.16.0 release)